pay-api/src/pay_api/resources/v1/account.py

```diff
@@ -137,7 +133,6 @@ def put_account(account_number: str):
 
 @bp.route('/<string:account_number>', methods=['DELETE'])
 @cross_origin(origins='*')
-@_tracing.trace()
```

Comment:
I don't think these are used I'll check with Patrick, it's out of date too the library is depreciated